Backend/app/dashboard/Lib/src/Environment.hs

```diff
@@ -92,11 +96,17 @@ buildAppEnv authTokenCacheKeyPrefix AppCfg {..} = do
   esqDBReplicaEnv <- prepareEsqDBEnv esqDBReplicaCfg loggerEnv
   coreMetrics <- registerCoreMetricsContainer
   let modifierFunc = ("dashboard:" <>)
+  let nonCriticalModifierFunc = ("dashboard:non-critical:" <>)
```

Comment:
can you provide shorter key name? `d:n_c:`?
Trying to reduce key's memory footprint